apps/admin-portal/src/components/identity-providers/settings/authenticator-settings.tsx

```diff
@@ -16,20 +16,21 @@
  * under the License.
  */
 
-import { AlertLevels } from "@wso2is/core/models";
-import { addAlert } from "@wso2is/core/store";
 import { ContentLoader, Heading } from "@wso2is/react-components";
-import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
-import { useDispatch } from "react-redux";
-import { getFederatedAuthenticatorDetails, getFederatedAuthenticatorMeta, updateFederatedAuthenticator } from "../../api";
 import {
     FederatedAuthenticatorListItemInterface,
     FederatedAuthenticatorListResponseInterface,
     FederatedAuthenticatorMetaInterface,
     SupportedAuthenticators
-} from "../../models";
-import { AuthenticatorFormFactory } from "./forms";
+} from "../../../models";
+import { getFederatedAuthenticatorDetails, getFederatedAuthenticatorMeta, updateFederatedAuthenticator }
```

Comment:
```suggestion
import {
    getFederatedAuthenticatorDetails,
    getFederatedAuthenticatorMeta,
    updateFederatedAuthenticator
} from "../../../api";
```

Reply:
Fixed in cb34564693f1d730c6a791813685e95dd490c094.